.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto merge for GH Actions
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' && contains( github.event.pull_request.labels.*.name, 'github_actions') }}
+    steps:
+      - name: Enable auto-merge for Dependabot GitHub Actions PRs
+        run: gh pr merge --auto --squash "$PR_URL"
+        env:
+          PR_TITLE: ${{github.event.pull_request.title}}
```

Comment:
@MartinGroscheTT 
Because we discussed about the title vs url. I think the title should be correct one. Do we need a PR description with the URL?

Reply:
Not really but you need to fix the env reference `PR_URL`  @L14 to `PR_TITLE`